senders/pagerduty/init.go

```diff
@@ -0,0 +1,30 @@
+package pagerduty
+
+import (
+	"time"
+
+	"github.com/moira-alert/moira"
+)
+
+// Sender implements moira sender interface for pagerduty
+type Sender struct {
+	ImageStores  map[string]moira.ImageStore
+	imageStoreID string
+	logger       moira.Logger
+	frontURI     string
+	location     *time.Location
+}
+
+// Init loads yaml config, configures the pagerduty client
+func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+	sender.frontURI = senderSettings["front_uri"]
+	sender.imageStoreID = senderSettings["image_store"]
+	if sender.imageStoreID == "" {
+		logger.Warningf("cannot read image_store from the config, will not be able to attach plot images to events")
```

Comment:
All our log entries start with a capital letter. Look at the remaining places in the code where you added logging and fix, please.